quickstarts/jboss-eap7-with-helm.yaml

```diff
@@ -0,0 +1,216 @@
+      summary:
+        failed: Try the steps again.
+        success: You viewed the code associated with the **eap74** deployment.
+      title: View the associated code
+    - description: >-
+        To view the build status of the JBoss EAP 7 application:
+
+        1. In the navigation menu, click [Topology]{{highlight qs-nav-topology}}.
+
+        1. In the Topology view, click **D eap74**.  
+           A side panel opens with detailed information about the application.
+
+        1. In the side panel, click the **Resources** tab.  
+           The **Builds** section shows all the details related to builds of the application.
+
+        The JBoss EAP 7 application is built in two steps:
+
+          - The first build configuration **eap74-build-artifacts** compiles and packages the Jakarta EE application, and creates a JBoss EAP server.
+            The application is run on this JBoss EAP server.
+
+            The build may take a few minutes to complete. The build progresses through various states such as **Pending**, **Running**, and **Complete**.
+            The build state is indicated by a relevant message.
+
+            When the build is complete, a checkmark and the following message is displayed: **Build #1 was complete**.
+
+            When the first build is complete, the second build starts.
+
+          - The second build configuration **eap74** puts the Jakarta EE deployment and the JBoss EAP server in a runtime image that contains only what is required to run the  application.
+
+            When the second build is complete, a checkmark and the following message are displayed: **Build #2 was complete**.
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: >-
+          The two builds for **eap74-build-artifacts** and **eap74** may take a few minutes to complete.
+
+          Verify the builds are complete:
+
+          - The message **Build #1 was complete** is displayed for the **eap74-build-artifacts** build configuration. Did this message appear?
+
+          - The message **Build #2 was complete** is displayed for the **eap74** build configuration. Did this message appear?
+      summary:
+        failed: Try the steps again.
+        success: Your build is complete.
+      title: View the Build status
+    - description: >-
+        To view the pod status:
+
+        1. In the navigation menu, click [Topology]{{highlight qs-nav-topology}}.
+
+        1. In the **Topology** view, click **D eap74**.  
+           A side panel opens with detailed information about the application.
+
+        1. In the **Details** tab, hover over the pod to see the pod status in a tooltip.
+
+            - Inside the pod circle, it displays the number of pod.
+            - The color of the pod circle indicates the pod status:
+              Light blue = **Pending**, Blue = **Not Ready**, Dark blue = **Running**
```

Comment:
Just thinking from an accessibility pov, is there something we can say here to avoid using "light blue, blue, and dark blue"? I think blue is one of the common color blindness colors to be aware of.

Reply:
When we did our user survey, this was reported. We did not find other good way to tell to the users what to look for.

We are consistent with OpenShift doc https://docs.openshift.com/container-platform/4.9/applications/odc-viewing-application-composition-using-topology-view.html#odc-interacting-with-applications-and-components_viewing-application-composition-using-topology-view but went further by mentioning different shades (light, normal, dark) so that someone colorblind might still see the difference.

Reply:
The tooltip also provides the pod status so the information is still accessible for colorblind people. 

<img width="422" alt="Screenshot 2023-05-17 at 14 31 59" src="https://github.com/openshift/console-operator/assets/6193/25da36ec-e581-49d5-b133-715641be789f">

When I look at the Quarkus quickstart, they only mention https://github.com/openshift/console-operator/blob/6fcb04d2ab155fd2b6282f69aa1581f7999da315/quickstarts/quarkus-with-s2i.yaml#L136 that I found even more confusing: which "blue" is it?